### ISSUE TITLE
Compilation error on a 32-bit operating system

### ISSUE THREAD

**Issue by hjx44599** [juicedata/juicefs#5301]
**What happened**: When I tried to compile the client on a 32-bit operating system, I encountered an integer overflow error. Is there a plan to support running on a 32-bit system?

![图片](https://github.com/user-attachments/assets/d8ffc52d-fddf-4fab-8d28-086e62419036)

**What you expected to happen**: "I hope to be able to use this excellent software on a 32-bit system."

**How to reproduce it (as minimally and precisely as possible)**:  To reproduce the issue, simply perform the compilation operation on a 32-bit Windows 10 system.

**Anything else we need to know?**
It appears that the error is occurring in the "consistenthash" and "tikv" dependencies referenced in the project.
**Environment**:
- JuiceFS version (use `juicefs --version`) or Hadoop Java SDK version: main 
- Cloud provider or hardware configuration running JuiceFS:   Deploy it myself.
- OS (e.g `cat /etc/os-release`):   windows10  32-bit

- Kernel (e.g. `uname -a`): 
- Object storage (cloud provider and region, or self maintained):
- Metadata engine info (version, cloud provider managed or self maintained):
- Network connectivity (JuiceFS to metadata engine, JuiceFS to object storage):
- Others:

**Comment by jiefenghuang** [comment]
There’s no plan to support it for now.

**Comment by davies** [comment]
@hjx44599 Can you try to build with `-tags notikv` ?

**Comment by hjx44599** [comment]
> [@hjx44599](https://github.com/hjx44599) Can you try to build with `-tags notikv` ?

Yes, when I don't need to use TiKV in my environment, I can package the 32-bit program through this command and run it successfully. Thank you very much for the solution you provided.
